src/mono/wasm/wasm.proj

```diff
@@ -108,17 +108,50 @@
     <Error Text="Failed to parse emcc version, and hash from the full version string: '$(_EmccVersionRaw)'"
            Condition="'$(_EmccVersion)' == '' or '$(_EmccVersionHash)' == ''" />
 
+    <!-- keep in sync with src\mono\wasm\build\WasmApp.Native.targets -->
+    <ItemGroup>
```

Comment:
Would it be useful to have a property `$(WasmSkipDefaultExportedRuntimeMethods)`, and one for runtime functions?

Reply:
Probably not